src/acceptance-testing/index.md

```diff
@@ -4,7 +4,7 @@ Acceptance testing ensures OP Stack networks are feature-complete, reliable, and
 
 The Platforms team will compile a Release Readiness Process (RRP) document, which will outline how to acceptance test devnets. This will include a list of tests to run - the Release Readiness Checklist (RRC). They will originally be run manually, but we'll automate them over time.
 
-By automating validation and enforcing quality gates, we reduce risk and increase confidence in releases. Much of this is facilitated by a new tool, op-acceptor, which can run standard Go tests against OP Stack networks and track that network’s readiness for promotion. Acceptance testing is a prerequisite for networks to promote from Alphanet, to Betanet, to Testnet.
+By automating validation and enforcing quality gates, we reduce risk and increase confidence in releases. Much of this is facilitated by a new tool, op-acceptor, which can run standard Go tests against OP Stack networks and track that network's readiness for promotion. Acceptance testing is a prerequisite for networks to promote from Alphanet, to Betanet, to Testnet.
```

Comment:
To clarify: does this mean that acceptance tests block alphanets or betanets? The way this reads, it seems like they block promotion to betanet but I want ot be sure.

Reply:
The intent was that it should block both/all. But the more advanced deployment stages may have more rigorous tests.